posts/2022/12-17-first-dev-job.md

```diff
@@ -0,0 +1,105 @@
+
+6. Somewhat related to the previous point: tell your team lead/manager what you need. It is part of their job to support you, and if you are upfront about what you need, you make their job easier.
+
+
+# 6. Work-life boundaries or the curse of notifications
+
+
+This one was a hard lesson to learn. I knew that I could only work so many hours per day. I'm an early morning person who tends to fate later in the afternoon. I'm not productive anymore. There is no good reason for me to work until late. Better to rest and be fresh in the morning. Sounds reasonable. But it's so hard!
+There were many days where I stayed on the computer much longer than I had originally intended. And what made matters worse: I had all my notifications on. So even after I had switched off the computer for the night, I would still see if somebody messaged me on Slack or would get an email notification about a code review on GitHub. This didn't make for very relaxing evenings and/or weekends. I wish I could say that I have this one figured out completely. I'm getting there, but I'm still relapsing sometimes. If you want to be smarter than me, follow these seemingly simple tips from the get-go:
+
+1.  Set up your GitHub notifications: avoid having GitHub notifications sent to you by email. Instead, set aside specific times during the day when you check your notifications directly on GitHub. 
+
+2.  Set up your Slack notifications so you don't get email notifications outside your working hours.
+
+3.  Remove Slack from your phone. Trust me. I thought for the longest time that I could have Slack on a separate screen and would be able to resist the temptation to check my work Slack. Spoiler: it didn't work at all. 
+
+4.  Don't work passed the point of exhaustion. When you are getting to the point where writing code feels like walking through molasses, stop. Make notes, so you know where to pick up in the morning and shut off the computer. You're done. Let it go!
+
+5. Don't let FOMO ruin your boundaries. If you have flexible working hours and are getting your work done, there is no reason to adjust your work-life patterns to those of your teammates. If they decide to work on the weekend, then that's their choice. It does not have to impact you at all. Special tip: if you are prone to FOMO, it REALLY helps to stay away from Slack and your work GitHub repo on the weekend! 
+
+
+# Final thoughts
+
+
+I hope some of these tips are helpful to you. I will heed my own advice in the future. Unfortunately, I no longer have a job as I am writing this. The CEO of the company I was working for had not been honest about the funding situation. As a result, the whole team was let go a few days ago. 
+
+
+
+
```

Comment:
I think that this is your most interesting article yet from my perspective. I think many others will agree too.

Reply:
Thank you for these kind words :)